examples/40_advanced/example_pandas_train_test.py

```diff
@@ -1,26 +1,29 @@
 # -*- encoding: utf-8 -*-
 """
 ===============================
-Test and Train data with Pandas
+Performance-over-time plot
 ===============================
+This example shows, how to use the *performance_over_time_* attribute to plot the performance
+over train time.  *performance_over_time_* can contain multiple metrics within a pandas dataframe, namely:
+
+    - ensemble_optimization_score
+    - ensemble_test_score
+    - single_best_optimization_score
+    - single_best_test_score
+    - single_best_train_score
 
 *auto-sklearn* can automatically encode categorical columns using a label/ordinal encoder.
 This example highlights how to properly set the dtype in a DataFrame for this to happen,
 and showcase how to input also testing data to autosklearn.
+
 The X_train/y_train arguments to the fit function will be used to fit the scikit-learn model,
 whereas the X_test/y_test will be used to evaluate how good this scikit-learn model generalizes
 to unseen data (i.e. data not in X_train/y_train). Using test data is a good mechanism to measure
 if the trained model suffers from overfit, and more details can be found on `evaluating estimator
 performance <https://scikit-learn.org/stable/modules/cross_validation.html#cross-validation>`_.
-This example further highlights through a plot, the best individual models found by *auto-sklearn*
-through time (under single_best_optimization_score/single_best_test_score's legend).
-It also shows the training and test performance of the ensemble build using the best
-performing models (under ensemble_optimization_score and ensemble_test_score respectively).
-
-There is also support to manually indicate the feature types (whether a column is categorical
```

Comment:
Can you please also preserve this message, that gives an intuition about the feature types?